tests/seq-properties.hs

```diff
@@ -540,6 +542,10 @@ prop_sortBy xs =
     toList' (sortBy f xs) ~= Data.List.sortBy f (toList xs)
   where f (x1, _) (x2, _) = compare x1 x2
 
+prop_sortOn :: Seq (OrdA, B) -> Bool
```

Comment:
Why not `Fun A OrdB -> Seq A -> Bool`? That seems to get to the point more obviously.